functests/5_latency_testing/latency_testing.go

```diff
@@ -97,6 +101,7 @@ type latencyTest struct {
 }
 
 var _ = Describe("Run tests of latency measurement tools with different values of latency environment variables", func() {
+	format.MaxLength = 0
```

Comment:
where is this used?

Reply:
this is limiting the gomega output presentation by default to 4000 chars. we set it to 0 to disable this feature. 
check commit https://github.com/openshift-kni/performance-addon-operators/pull/964/commits/34614661dc46f5b4e1d7e8d50031d9dd784c490b